ScnSoft-Education/LeetCodeTraining.Tests/StringToInteger8.Tests.cs

```diff
@@ -0,0 +1,18 @@
+﻿using LeetCodeTraining.StringToInteger8;
+namespace LeetCodeTraining.Tests
+{
+    public class MyAtoiTests
+    {
+        [Theory]
+        [InlineData("42", 42)]
+        [InlineData("   -42", -42)]
+        [InlineData("4193 with words", 4193)]
+        [InlineData("words and 987", 0)]
```

Comment:
add unit case when we have string value without digits - "If no digits were read, then the integer is 0."